mautic/api-library
# Additional Api improvements, basic auth

These changes rely on PR #70 being in place first.

Includes a fix for the Data class plus an initial test case and replacement of deprecated methods.

**Basic Authentication Support Added**
I've added support for the BasicAuth functionality now in Mautic. See the update to the readme for usage instructions plus I've updated the local.config.php.dist file with instructions. 

To Test this PR:
1. See #70 and follow instructions. Install this PR
2. Run the included test for the data class `phpunit tests/Api/DataTest.php`
3. Run the BasicAuth test case by running `phpunit tests/Api/Auth/BasicAuthTest.php`
